### PR TITLE
Added upcoming FJLabs Polaris PCB's

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Added upcoming FJLabs Polaris PCB's. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — CHANGES_REQUESTED** [review]
Please revert the changes to the submodules. 

You may need to run `make git-submodule` and then commit, to do so

**Comment by mrnoisytiger** [comment]
@drashna Submodules reverted.

Steps:

1. Checkout branch
2. `cd <qmk base dir>`
3. `make git-submodule`

```
QMK Firmware 0.12.15
Synchronizing submodule url for 'lib/chibios'
Synchronizing submodule url for 'lib/chibios-contrib'
Synchronizing submodule url for 'lib/googletest'
Synchronizing submodule url for 'lib/lufa'
Synchronizing submodule url for 'lib/printf'
Synchronizing submodule url for 'lib/ugfx'
Synchronizing submodule url for 'lib/vusb'
Submodule path 'lib/chibios': checked out '313416b8fda90d9973a749a0a35970956852c286'
Submodule path 'lib/chibios-contrib': checked out 'e3a3a24047717af33b098266e11c2e4e5102206a'
```

4. `cd <keyboard dir>`
5. Make random small change
6. `git add .`
7. `git commit -m"___"`
8. `git push`

**Comment by drashna** [comment]
Awesome! Though there is a flag to allow empty commits, too.

**Comment by mrnoisytiger** [comment]
> Awesome! Though there is a flag to allow empty commits, too.

This is news to me. 😂  Thanks for the info!

**Comment by mrnoisytiger** [comment]
Any chance I could get review on this? Submodules should be fixed.

**Comment by mrnoisytiger** [comment]
@drashna  Could I get the dependency tag removed on this guy? Thanks!

**Review by drashna — APPROVED** [review]

File: keyboards/fjlabs/polaris/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Polaris Hotswap and Solder
+ 
+Make example for this keyboard (after setting up your build environment):
+
+    make fjlabs/polaris:default
+
+Alternatively, if using VIA, use the following example:
+    make fjlabs/polaris:via
+
```
Comment (fauxpark) [inline keyboards/fjlabs/polaris/readme.md:13]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Submodule changes still need to be undone.

**Comment by mrnoisytiger** [comment]
This should do it...... 
```
make git-submodule
git commit --allow-empty -m "Fix submodules"
git push
```

**Comment by fauxpark** [comment]
Nope. You may need to rebase instead.

**Comment by mrnoisytiger** [comment]
Closing to recreate new branch and destroy current.

**Pull request closed without merging**